### ISSUE TITLE
Improve support for placementStrategy and placementConstraints

### ISSUE THREAD

**Issue by efeller** [Shippable/support#4406]
We are trying to use placementStrategy field in dockerOptions and it does not seem to work. No problems in the build:

https://app.shippable.com/github/zenreach/jobs/sightings-zp-phantom/builds/5b280a17fbb9510700f4ce9e/console

However, the resulting service on AWS ECS does not have a task placement strategy set. Under the deployments section it simply says:

Strategy No strategies

**Comment by efeller** [comment]
This might be working. However, it needs a destroy of the job on ECS and a redeploy via shippable with a reset. Can you please fix it such that I can add placementStrategy without a job delete and redeploy?

Thanks

**Comment by trriplejay** [comment]
@efeller yeah, this is the behavior of ECS itself.  Internally, we would still have to delete/redeploy the service. Is that the behavior you would want? (similar to what happens when changing a load balancer)

**Comment by efeller** [comment]
@trriplejay Yea, it would be fine if shippable did the delete/redeploy for me. Thanks.
